cocos/serialization/ccon.ts

```diff
@@ -230,6 +229,6 @@ export class BufferBuilder {
     }
 }
 
-legacyCC.internal.parseCCONJson = parseCCONJson;
-legacyCC.internal.decodeCCONBinary = decodeCCONBinary;
-legacyCC.internal.CCON = CCON;
+cclegacy.internal.parseCCONJson = parseCCONJson;
+cclegacy.internal.decodeCCONBinary = decodeCCONBinary;
```

Comment:
Why rename `legacyCC` to `cclegacy`?

Reply:
In cc.d.ts, it is exported as cclegacy. So also change engine internal usage too.